RedemptionQuick/sets/carddata.txt

```diff
@@ -803,10 +803,10 @@ False Teachers (L)	Main	False_Teachers_(UL)	Evil Character	Gray	6	6				Uncommon
 False Teachers (UL)	Main Unlimited	False_Teachers_(UL)	Evil Character	Gray	6	6				Uncommon	Jude 4	
 False Wisdom	Prophets	False_Wisdom_(Pr)	EE	Gray					Holder may look at rescuer's hand.	Common	Isaiah 47:10	
 False Witnesses	10th Anniversary	False_Witnesses_(G)	EE	Black	2	2			If used by a Sadducee, capture a hero.	Deck	Matthew 26:59	
-Family Bond	J	Family_Bond_(J)	GE	Gold	2	2			Negate and discard the last evil enhancement played this battle. 	Deck	Ruth 1:16	
+Family Bond	J	Family_Bond_(J)	GE	White/Gold	2	2			Negate and discard the last evil enhancement played this battle. 	Deck	Ruth 1:16	
```

Comment:
Should by Gold/White  for consistency

Reply:
Found that some of the cards had their 1st brigade written first, their 2nd afterwards etc. and for some other cards it was vice versa. Is there a pattern?

Reply:
Good then Evil
Alphabetical within good and evil.
It gives a consistent way for users to know how to search.